app/admin/dashboard.rb

```diff
@@ -31,19 +31,27 @@
         end
       end
 
-      column do
-        panel "Recent Posts" do
-          last_posts = Post.last(10).reverse
-          ul do
-            last_posts.map do |post|
-              title = post.title
-              title ||= title = 'Post made on ' + post.created_at.to_s
-              li link_to(title, admin_post_path(post))
+      if Post.all.length > 0
+        column do
+          panel "Recent Posts" do
+            last_posts = Post.last(10).reverse
+            ul do
+              last_posts.map do |post|
+                title = post.title
+                title ||= title = 'Post made on ' + post.created_at.to_s
+                li link_to(title, admin_post_path(post))
+              end
             end
           end
         end
       end
 
+        column do
+          panel "Analytics" do
+            para "Explore the GTHC data with SQL. Easily create charts and dashboards, and share them with other admins."
+            para link_to "View Blazer Analytics", blazer.root_path, class: 'button'
```

Comment:
@mafuvadze Fixed this in new commit. We were doing the right thing originally, but we forgot to access routes by directly calling the engine because the routes were being handled by the Blazer engine. The answer to this StackOverflow question is a good example:
https://stackoverflow.com/questions/30113070/accessing-engine-routes-from-rails-application-routes-url-helpers